### ISSUE TITLE
rename 'dummy' directory to 'hal'

### ISSUE THREAD

**Issue by tballmsft** [microsoft/jacdac-ts#87]
the files in the "dummy" directory of "interfaces" really represent the hardware abstraction layer that JD relies on. Can we use a different name than "dummy"?  Perhaps "hal"?

**Comment by jamesadevine** [comment]
Are we supposed to be filing issues for other repositories here?

**Comment by jamesadevine** [comment]
I assume this is intended for jacdac-c?

**Comment by tballmsft** [comment]
Yes, this is for jacdac-c (and anywhere the structure is mirrored). Adding @pelikhan, @mmoskal. James, I think we should use one repo to collect issues, as much as possible.

**Comment by jamesadevine** [comment]
Sure, no problem with consolidation. Can we please use labels to indicate the affected repos? I had to check whether jacdac-ts had dummy interfaces.
